source/_components/delijn.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "De Lijn"
+description: "Instructions on how to integrate De Lijn (Flemish public transport company) departure times into Home Assistant."
+date: 2019-05-23 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_release: "0.94"
+ha_category: 
```

Comment:
it's also a `sensor` 😉

Reply:
Makes sense :) Thanks for the suggestions